samples/highcharts/sonification/series-earcon/demo.js

```diff
@@ -1,4 +1,12 @@
-Highcharts.chart('container', {
+Highcharts.setOptions({
+    lang: {
+        accessibility: {
+            chartContainerLabel: 'Click series to sonify chart.'
```

Comment:
Actually this one I would keep, but change, since the title is very similar to other charts. Maybe `"Earcons on series. Highcharts interactive chart."`?